editor/plugins/tiles/tile_map_editor.cpp

```diff
@@ -398,6 +402,10 @@ void TileMapEditorTilesPlugin::_update_scenes_collection_view() {
 			scene_tiles_list->select(item_index, false);
 		}
 	}
+	if (scene_tiles_list->get_item_count() == 0) {
+		scene_tiles_list->add_item(TTR("The selected scene collection source has no scenes. Add scenes in the TileSet bottom tab."));
```

Comment:
This is a bit hacky TBH, message should likely be displayed with a Label. It's looks a bit weird you can hover/select such a message.

Reply:
It can't be selected. It'd disabled.

Reply:
Yeah, but the hover effect is still a bit confusing about it IMO. It's not a big issue though, if you don't want to fix that right now I think the PR can be merged anyway.

Reply:
If it can be merged then please approve 🙃